src/NationalInstruments.Analyzers/Correctness/ApprovedNamespaceAnalyzer.cs

```diff
@@ -0,0 +1,252 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Text;
+using NationalInstruments.Analyzers.Properties;
+using NationalInstruments.Analyzers.Utilities;
+using NationalInstruments.Analyzers.Utilities.Extensions;
+
+namespace NationalInstruments.Tools.Analyzers.Namespaces
```

Comment:
Update the namespace. It doesn't match the layout of the rest of the files in the single assembly now. There are currently two main namespaces: NationalInstruments.Analyzers.Correctness and NationalInstruments.Analyzers.Style

We need to figure out if you want a third, but we should match the hierarchy and ensure that the folder structure matches. Currently this analyzer is inside Correctness

Reply:
Fixed it.